api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -496,15 +496,15 @@ public ResponseEntity<Workspace> updateWorkspace(
         workspaceService.updateWorkspaceBillingAccount(
             dbWorkspace, request.getWorkspace().getBillingAccountName());
       } catch (ServerErrorException e) {
-        new ServerErrorException("Could not update the workspace's billing account", e);
+        throw new ServerErrorException("Could not update the workspace's billing account", e);
       }
     }
 
     try {
-      dbWorkspace.setLastModifiedBy(userProvider.get().getUsername());
```

Comment:
updated saveWithLastModified() so that it now records the last-modified-by user as well as the last modified time.

So this call is no longer necessary.